news.md

```diff
@@ -3,7 +3,20 @@ title: News
 ---
 ## News
 
-### Jython 2.7.2 beta (November 2019)
+### Jython 2.7.2 beta (v2.7.2b3 February 2020)
+
+A further beta release is now available for Jython 2.7.2 at
+[Maven Central](https://search.maven.org/search?q=g:org.python).
+It is built and tested with Java 8 and tested against Java 11.
+This fixes bugs raised in response to v2.7.2b2 and updates certain JARs.
+Although past predictions of releases very reliable, we predict a release candidate 24 February.
```

Comment:
This sentence looks a bit strange. And a release candidate in 24 February sounds great!

Reply:
Grrr. Always seem to mess up details like this. :/